result/yoda.js

```diff
@@ -0,0 +1,19 @@
+
+/*eslint no-multi-spaces: "error"*/
+
+var a =  1;
+var a =  1;
+var a =  1;
```

Comment:
__[eslint]__
* Error - 'a' is already defined (no-redeclare)
* Error - Multiple spaces found before '1'. (no-multi-spaces)